crates/eips/src/eip1559/basefee.rs

```diff
@@ -7,17 +7,17 @@ use crate::eip1559::constants::{
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct BaseFeeParams {
     /// The base_fee_max_change_denominator from EIP-1559
-    pub max_change_denominator: u64,
+    pub max_change_denominator: u128,
     /// The elasticity multiplier from EIP-1559
-    pub elasticity_multiplier: u64,
+    pub elasticity_multiplier: u128,
 }
 
 impl BaseFeeParams {
     /// Get the base fee parameters for Ethereum mainnet
     pub const fn ethereum() -> BaseFeeParams {
         BaseFeeParams {
-            max_change_denominator: DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR,
-            elasticity_multiplier: DEFAULT_ELASTICITY_MULTIPLIER,
+            max_change_denominator: DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR as u128,
```

Comment:
should we change the constant types?

Reply:
Ideally yes, but I think if users would like to use constants in their applications; It's smoother for them to use `as u128` and expand to a large type than use `try_from().into()` to shrink to a smaller.

Reply:
Moreover, imo, its better to expand the type in places where its absolutely needed and keep it as is where isn't required.